frontend/src/types.ts

```diff
@@ -605,38 +605,68 @@ export enum ProgressionStep {
   SERVER_REQUESTED = 'SERVER_REQUESTED',
   POD_CREATED = 'POD_CREATED',
   POD_ASSIGNED = 'POD_ASSIGNED',
+  POD_PROBLEM = 'POD_PROBLEM',
```

Comment:
```suggestion
  POD_PROBLEM = 'POD_PROBLEM',
  POD_CREATED = 'POD_CREATED',
  POD_ASSIGNED = 'POD_ASSIGNED',
```
A pod problem will prevent the pod from existing most likely... scaling, not enough resources, etc -- all of which prevent a pod from existing. It's an awkward case because there is no "we are starting to create a pod" event like there is for "starting to pull an image"